src/Elasticsearch/Endpoints/Scroll.php

```diff
@@ -36,70 +31,38 @@ public function setBody($body)
     }
 
     /**
-     * @return array
-     */
-    public function getBody()
-    {
-        return $this->body;
-    }
-
-    /**
-     * @param string $scroll
-     *
      * @return $this
      */
-    public function setScroll($scroll)
+    public function setScrollId(?string $scrollId): Scroll
     {
-        if (isset($scroll) !== true) {
-            return $this;
+        if ($scrollId !== null) {
+            $this->scrollId = $scrollId;
         }
-
-        $this->body['scroll'] = $scroll;
-
         return $this;
     }
 
-    /**
-     * @param string $scroll_id
-     *
-     * @return $this
-     */
-    public function setScrollId($scroll_id)
+    public function getURI(): string
     {
-        if (isset($scroll_id) !== true) {
-            return $this;
-        }
+        $scrollId = $this->scrollId ?? null;
 
-        $this->body['scroll_id'] = $scroll_id;
+        if (isset($scrollId)) {
+            return "/_search/scroll/$scrollId";
```

Comment:
I can't seem to find where this is documented for es 7.x. Tried the 7.0 release version with es 7.2 but the scrollId is not picked up by elasticsearch.

Reply:
The API specification for `scroll` endpoint, using ES 7.0.0, is reported [here](https://github.com/elastic/elasticsearch/blob/v7.0.0/rest-api-spec/src/main/resources/rest-api-spec/api/scroll.json#L7). If you are using ES 7.2 the `/_search/scroll/$scrollId` has been deprecated, see [here](https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/src/main/resources/rest-api-spec/api/scroll.json#L8-L13). 
I'll release elasticsearch-php 7.1 soon and I need to manage some deprecated paths for ES 7.2, that will be released soon (current master of [elastic/elasticsearch](https://github.com/elastic/elasticsearch)).